### DISCUSSION TITLE
PaddleOCR2.7.5 训练到几百回合出现loss>1问题

### DISCUSSION THREAD

**Discussion by 18325391772** [PaddlePaddle/PaddleOCR#13306] [Q&A]
![1720147935352](https://github.com/PaddlePaddle/PaddleOCR/assets/43359721/41d332ec-1315-4a71-b594-4508d704b1f3)

报错代码：
class DiceLoss(nn.Layer):
    def __init__(self, eps=1e-6):
        super(DiceLoss, self).__init__()
        self.eps = eps

    def forward(self, pred, gt, mask, weights=None):
        """
        DiceLoss function.
        """

        assert pred.shape == gt.shape
        assert pred.shape == mask.shape
        if weights is not None:
            assert weights.shape == mask.shape
            mask = weights * mask
        intersection = paddle.sum(pred * gt * mask)

        union = paddle.sum(pred * mask) + paddle.sum(gt * mask) + self.eps
        loss = 1 - 2.0 * intersection / union
        assert loss <= 1
        return loss

**Comment by GreatV** [comment]
升级到2.8.0试试，照理说不会出现loss>1的情况

**Comment by 2013358072** [comment]
我也遇到了这个问题

**Comment by 1005cyxin** [comment]
参数设置问题/没安装paddleocr
